packages/terra-demographics-banner/src/_sharedObjects.jsx

```diff
@@ -1,23 +1,46 @@
 import classNames from 'classnames/bind';
 import React from 'react';
+import VisuallyHiddenText from 'terra-visually-hidden-text';
 import styles from './DemographicsBanner.module.scss';
 
 const cx = classNames.bind(styles);
 
 // eslint-disable-next-line react/prop-types
-const DemographicsBannerValue = ({ label, value }) => (
-  <span className={cx('value')}>
-    { label && <span className={cx('value-label')}>{`${label}:`}</span> }
-    <b>{value}</b>
-  </span>
-);
+const DemographicsBannerValue = ({ label, value, abbrTitle }) => {
+  let valueLabelContent;
+
+  if (abbrTitle) {
+    valueLabelContent = (
+      <span className={cx('value-label')}>
+        <abbr className={cx('abbreviation')} title={abbrTitle} aria-hidden="true">
+          {`${label}:`}
+        </abbr>
+      </span>
+    );
+  } else if (label) {
+    valueLabelContent = (
+      <span className={cx('value-label')}>
+        {`${label}:`}
+      </span>
+    );
+  }
+
+  return (
+    <span className={cx('value')}>
+      {abbrTitle && (<VisuallyHiddenText text={abbrTitle} />)}
+      {valueLabelContent}
+      <b>{value}</b>
```

Comment:
Should we use css styling here instead of the `<b/>` tag

Reply:
yeah, we'll prob need to convert this to a themeable font-weight: https://github.com/cerner/terra-core/issues/1897